openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/ServiceIngestion.spec.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ *  Copyright 2024 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import test from '@playwright/test';
+import MetabaseIngestionClass from '../../support/entity/ingestion/AirflowIngestionClass';
+import BigQueryIngestionClass from '../../support/entity/ingestion/BigQueryIngestionClass';
+import KafkaIngestionClass from '../../support/entity/ingestion/KafkaIngestionClass';
+import MlFlowIngestionClass from '../../support/entity/ingestion/MlFlowIngestionClass';
+import MysqlIngestionClass from '../../support/entity/ingestion/MySqlIngestionClass';
+import PostgresIngestionClass from '../../support/entity/ingestion/PostgresIngestionClass';
+import RedshiftWithDBTIngestionClass from '../../support/entity/ingestion/RedshiftWithDBTIngestionClass';
+import S3IngestionClass from '../../support/entity/ingestion/S3IngestionClass';
+import SnowflakeIngestionClass from '../../support/entity/ingestion/SnowflakeIngestionClass';
+import SupersetIngestionClass from '../../support/entity/ingestion/SupersetIngestionClass';
+import { redirectToHomePage } from '../../utils/common';
+import { settingClick } from '../../utils/sidebar';
+
+const services = [
+  S3IngestionClass,
+  MetabaseIngestionClass,
+  MysqlIngestionClass,
+  // Todo: need to skip for time being as AUT runs on argo, and airflow is not available
+  //   new AirflowIngestionClass(),
```

Comment:
I think based on isOSS flag we can now enable disable this test? WDYT?

Reply:
Done